### ISSUE TITLE
Problem with notebook example_HAR

### ISSUE THREAD

**Issue by ldv1** [GAMES-UChile/mogptk#13]
Hi,

there are a couple of problems with the notebook example_HAR, API changed ?

in particular

```
    channel.set_pred_range(start=0.0, end=8.3, n=200)
AttributeError: 'Data' object has no attribute 'set_pred_range'

```
and
```
    model.estimate_params('BNSE')
AttributeError: 'MOSM' object has no attribute 'estimate_params'

```
regards,

Laurent

**Comment by tdewolff** [comment]
Hi Laurent, thanks for raising the issue. Yes, unfortunately (some of) the examples are still behind on the recent changes we made. I will make a priority of this! Mean while, the tutorials should be functioning well.

**Comment by tdewolff** [comment]
The examples should be fixed with 57c49aea56a59ead70db4e38e9334a330c2ce896, 07414ec709c4eb88d578efde6549925e7ebc2364, and 97db2a50c67e8639b55b159b7aa3e6244ef77cf9. Let me know if you encounter problems still!
